pkg/koordlet/runtimehooks/config.go

```diff
@@ -77,31 +77,34 @@ var (
 )
 
 type Config struct {
-	RuntimeHooksNetwork       string
-	RuntimeHooksAddr          string
-	RuntimeHooksFailurePolicy string
-	RuntimeHookConfigFilePath string
-	RuntimeHookHostEndpoint   string
-	RuntimeHookDisableStages  []string
-	FeatureGates              map[string]bool // Deprecated
+	RuntimeHooksNetwork           string
+	RuntimeHooksAddr              string
+	RuntimeHooksFailurePolicy     string
+	RuntimeHooksCallFailurePolicy string
+	RuntimeHookConfigFilePath     string
+	RuntimeHookHostEndpoint       string
+	RuntimeHookDisableStages      []string
+	FeatureGates                  map[string]bool // Deprecated
 }
 
 func NewDefaultConfig() *Config {
 	return &Config{
-		RuntimeHooksNetwork:       "unix",
-		RuntimeHooksAddr:          "/host-var-run-koordlet/koordlet.sock",
-		RuntimeHooksFailurePolicy: "Ignore",
-		RuntimeHookConfigFilePath: system.Conf.RuntimeHooksConfigDir,
-		RuntimeHookHostEndpoint:   "/var/run/koordlet/koordlet.sock",
-		RuntimeHookDisableStages:  []string{},
-		FeatureGates:              map[string]bool{},
+		RuntimeHooksNetwork:           "unix",
+		RuntimeHooksAddr:              "/host-var-run-koordlet/koordlet.sock",
+		RuntimeHooksFailurePolicy:     "Ignore",
+		RuntimeHooksCallFailurePolicy: "Ignore",
+		RuntimeHookConfigFilePath:     system.Conf.RuntimeHooksConfigDir,
+		RuntimeHookHostEndpoint:       "/var/run/koordlet/koordlet.sock",
+		RuntimeHookDisableStages:      []string{},
+		FeatureGates:                  map[string]bool{},
 	}
 }
 
 func (c *Config) InitFlags(fs *flag.FlagSet) {
 	fs.StringVar(&c.RuntimeHooksNetwork, "runtime-hooks-network", c.RuntimeHooksNetwork, "rpc server network type for runtime hooks")
 	fs.StringVar(&c.RuntimeHooksAddr, "runtime-hooks-addr", c.RuntimeHooksAddr, "rpc server address for runtime hooks")
 	fs.StringVar(&c.RuntimeHooksFailurePolicy, "runtime-hooks-failure-policy", c.RuntimeHooksFailurePolicy, "failure policy for runtime hooks")
+	fs.StringVar(&c.RuntimeHooksCallFailurePolicy, "runtime-hooks-call-failure-policy", c.RuntimeHooksCallFailurePolicy, "stop running other hooks once someone failed")
```

Comment:
how about naming as `RuntimeHooksPluginFailurePolicy`, since it represents "stop running other plugins once someone failed"

Reply:
fixed